templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -241,52 +241,58 @@
   </table>
 
   {if !$contributionMode}
-    <fieldset class="payment-details_group">
```

Comment:
oh bugger I was wrong - this is all about contribution Mode - yuck. will rethink